Description: Port to pari 2.7 API
Author: Luca Falavigna <dktrkranz@debian.org>
        Tobias Hansen <thansen@debian.org>
Bug-Debian: http://bugs.debian.org/635506
            http://bugs.debian.org/743327

--- a/src/Lcommandline_elliptic.cc
+++ b/src/Lcommandline_elliptic.cc
@@ -121,11 +121,11 @@
 
 
     F = cgetg(6, t_VEC);
-    F[1] = lgeti(BIGDEFAULTPREC);
-    F[2] = lgeti(BIGDEFAULTPREC);
-    F[3] = lgeti(BIGDEFAULTPREC);
-    F[4] = lgeti(BIGDEFAULTPREC);
-    F[5] = lgeti(BIGDEFAULTPREC);
+    F[1] = (long)cgeti(BIGDEFAULTPREC);
+    F[2] = (long)cgeti(BIGDEFAULTPREC);
+    F[3] = (long)cgeti(BIGDEFAULTPREC);
+    F[4] = (long)cgeti(BIGDEFAULTPREC);
+    F[5] = (long)cgeti(BIGDEFAULTPREC);
 
     //gaffsg(a1,(GEN) F[1]);
     //gaffsg(a2,(GEN) F[2]);
@@ -139,7 +139,7 @@
     gaffect(strtoGEN(a4), (GEN) F[4]);
     gaffect(strtoGEN(a6), (GEN) F[5]);
 
-    E = initell(F,BIGDEFAULTPREC);
+    E = ellinit(F, NULL, BIGDEFAULTPREC);
 
     C=globalreduction(E);
 
